Source/utils/display.cpp

```diff
@@ -68,7 +68,7 @@ void AdjustToScreenGeometry(int width, int height)
 	gnScreenHeight = height;
 
 	borderRight = 64;
-	if (gnScreenWidth % 4) {
+	if ((gnScreenWidth % 4) != 0) {
 		// The buffer needs to be devisable by 4 for the engine to blit correctly

```

Comment:
divisible :P

Reply:
Took me a while to figure out what you where talking about, why not use the suggestion feature :D
```suggestion
		// The buffer needs to be divisible by 4 for the engine to blit correctly
```

Reply:
One has to be aware of the feature to use the feature!

Now that I'm aware of it, I'll use it :)